src/main/java/org/fundacionjala/coding/christian/Kata1.java

```diff
@@ -0,0 +1,30 @@
+package org.fundacionjala.coding.christian;
+
+import java.util.StringJoiner;
+
+/**
+ * Powered by Christian.
+ */
+
+public class Kata1 {
+
+    public static final int LIMIT = 5;
+
+    /**
+     * @param words this is words for Spin Words.
+     * @return result.
+     */
+    public String spinWords(String words) {
+
+        StringJoiner join = new StringJoiner(" ");
+        String[] arr = words.split(" ");
+        for (String c : arr) {
+            join = c.length() >= LIMIT
+                    ? join.add(new StringBuilder(c).reverse()) : join.add(c);
+        }
+
+        System.out.println(join);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)